app/views/Overlap.js

```diff
@@ -38,6 +39,8 @@ const base64 = RNFetchBlob.base64
 
 const public_data = "https://docs.google.com/spreadsheets/d/1itaohdPiAeniCXNlntNztZ_oRvjh0HsGuJXUJWET008/export?format=csv"
 const show_button_text = "Show Me Trace Overlap";
+const overlap_true_button_text = "Trace Overlap Found";
```

Comment:
We need to make these translatable strings.

Reply:
See the ```languages.t('label.overlap')``` below for an example.  The strings come from ../lcoales/us/overlap.json

Reply:
Translation wasn't set up for the button text at all, but I can go ahead and add it as part of this PR